### DISCUSSION TITLE
Demande de confirmation, combat ⚡️non régénèrés

### DISCUSSION THREAD

**Discussion by OrionYp** [Crownicles/Crownicles#2511] [Nouvelles idées]
Ajouter une demande de confirmation lorsque l'on lance ou accepte une demande de combat classé et que notre énergie n'est pas à 100%.
Comme ça on ne pourras plus dire qu'on n'a pas vu qu'on n'avait pas fini de regen.

**Comment by StrangerIM** [comment]
J'ai comme l'impression que c'est visé ça 😭 mais bonne idée ça évitera de faire un fight pour rien

**Comment by Ravenclaw-l** [comment]
On ne devrait pas avoir la possibilité de combattre sans être regen à 100%.

**Reply by OrionYp** [reply]
C'est une autre suggestion, moi je pense qu'on devrait laisser cette possibilité pour les joueurs de haut lvl qui ont longtemps délaissé le classé et qui peuvent enchaîner plusieurs combats sans regen pour monter en gloire
